libraries/Microsoft.Bot.Builder/Bot.cs

```diff
@@ -98,6 +101,17 @@ public Bot Use(Middleware.IMiddleware middleware)
         {
             var context = new BotContext(this, reference);
             await RunPipeline(context, proactiveCallback).ConfigureAwait(false);
-        }    
+        }
+
+        public static HttpClient GetHttpClientInstance()
+        {
+            if (_httpClient == null)
```

Comment:
This code is not thread safe. Given the highly concurrent nature of the WebAPI stack under which this code will normally be called, this code will need to be properly initialized.